### PR TITLE
Remove references to cancelled Browzine application

### PR DESCRIPTION
**Pull request by redlibrarian**
## Context

We cancelled our Browzine subscription as of April 30, 2020.

Related to # (issue) #1963 

## What's New

- Removed paragraph linking to Browzine in record views.
- Adjusted services spec and fixtures to reflect removal of Browzine from services.

### CODE REVIEW

File: spec/unit/promoted_services_spec.rb [hunk 1]
```diff
@@ -14,9 +14,9 @@
   end
 
   it 'should hold all the right fields' do
-    expect(service_document.id).to eq ['service001']
-    expect(service_document.title).to eq ['Browzine']
-    expect(service_document.url).to eq ['http://guides.library.ualberta.ca/browzine']
+    expect(service_document.id).to eq ['service002']
+    expect(service_document.title).to eq ['ERA (Education &amp; Research Archive)']
```
Comment (pgwillia) [inline spec/unit/promoted_services_spec.rb:18]:
```suggestion
    expect(service_document.title).to eq ['ERA (Education & Research Archive)']
```

**Comment by redlibrarian** [comment]
@pgwillia Thanks! I don't know how cutting releases is handled now - it's been a long time since I've done one, and I think that was before the "modernization" of the process. Can I cut the release from the master branch myself, once the PR is merged?

**Comment by pgwillia** [comment]
> 
> 
> @pgwillia Thanks! I don't know how cutting releases is handled now - it's been a long time since I've done one, and I think that was before the "modernization" of the process. Can I cut the release from the master branch myself, once the PR is merged?

Basically
- choose a version based on [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
- create a [PR like this one](https://github.com/ualbertalib/discovery/pull/1954) that bumps the [CHANGELOG version](https://github.com/ualbertalib/discovery/blob/master/CHANGELOG.md#unreleased) # and the [Application version](https://github.com/ualbertalib/discovery/blob/master/config/application.rb#L11) # 
- cut a [release on Github](https://github.com/ualbertalib/discovery/releases/new)

**Comment by redlibrarian** [comment]
👍

**Pull request merged**